guides/common/modules/proc_registering-hosts-for-conversion.adoc

```diff
@@ -0,0 +1,23 @@
+[id="registering-hosts-for-conversion_{context}"]
+
+= Registering Hosts for Conversion
+
+Use this procedure to register CentOS hosts to {Project}.
```

Comment:
see comment above about rhel derivatives.

Reply:
For beta only CentOS is suitable for conversion. Others will  be added later when supported.